qemu/tests/virtio_port_hotplug.py

```diff
@@ -31,32 +32,43 @@ def run(test, params, env):
 
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
+    os_type = params["os_type"]
     timeout = int(params.get("login_timeout", 360))
-    if params["os_type"] == "windows":
-        run_bg_test(test, params, vm)
-    for repeat in range(int(params.get("repeat_times", 1))):
-        repeat += 1
+    module = params.get("modprobe_module")
+    unload_module = params.get("unload_module", "yes")
+    if os_type == "windows":
+        driver_name = params["driver_name"]
+        session = vm.wait_for_login()
+        session = utils_test.qemu.windrv_check_running_verifier(
+            session, vm, test, driver_name)
+        session.close()
+    for port in params.objects("serials"):
         session = vm.wait_for_login(timeout=timeout)
-        module = params.get("modprobe_module")
-        if module:
+        if module and unload_module == 'yes':
```

Comment:
If i understand you correctly, `module = params.get("modprobe_module")` means need load module before test, and `unload_module` means need unload module when do unplug test.
Then before hotplug/unplug test:
```
if module:
    *load module*
```

Reply:
no, I want to set a parameter to control whether we need to operate module `virtio_console`. Do you have a suggestion for this parameter's name?

Reply:
Or we set the parameter `module = ""`, what do you think?